src/app/screens/LoginScreen.js

```diff
@@ -1,22 +1,50 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components'
 import { createGlobalStyle } from 'styled-components'
 
 //Component
 
-function LoginScreen(props) {
-    return (
-        
-        <div>
-           <GlobalStyle/>
-           <Form>
-               <Input type = "text" placeholder = "Username"/>
-               <Input type = "password" placeholder ="Password"/>
-               <Button type="submit">Login</Button>
-               <p>Not registered? <A href="#">Create an account</A></p>
-           </Form>
-        </div>
-    )
+function LoginScreen() {
+  const [userInfo, setInfo] = useState({userName: "", pass: ""});

```

Comment:
Its better if you keep it simple, and make multiple useStates for the scope. But this is ok for now, keep in mind